assets/base/i18n/en.json

```
@@ -296,6 +296,7 @@
   "general_settings.display": "Display",
   "general_settings.help": "Help",
   "general_settings.notifications": "Notifications",
+  "general_settings.report_problem": "",
```

I know is not from this PR but can you apply this?

```suggestion
  "general_settings.report_problem": "Report a problem",
```